src/content-layout/styles.scss

```diff
@@ -6,55 +6,71 @@
 @use '../internal/styles/tokens' as awsui;
 @use '../internal/generated/custom-css-properties/index.scss' as custom-props;
 
-/*
-Pass through the header and child content if not rendering in 
-visual refresh mode. The padding is still added to the header 
-in this scenario because otherwise the header and child 
-nodes will directly touch with no gap between them.
-*/
-.layout:not(.is-visual-refresh) {
-  display: contents;
-
-  > .background {
-    display: none;
-  }
-
-  > .header {
-    padding-block-end: awsui.$space-content-header-padding-bottom;
-  }
-}
-
-.layout.is-visual-refresh {
+.layout {
+  #{custom-props.$contentLayoutMaxContentWidth}: 0px;
+  #{custom-props.$contentLayoutMainGap}: 0px;
   display: grid;
-  grid-template-columns: minmax(0, 1fr);
-  grid-template-rows: auto #{awsui.$space-dark-header-overlap-distance} 1fr;
+  grid-template-columns: 0 0 1fr minmax(0, var(#{custom-props.$contentLayoutMaxContentWidth})) 1fr 0 0;
+  grid-template-rows: var(#{custom-props.$contentLayoutMainGap}) min-content min-content auto #{awsui.$space-dark-header-overlap-distance} 1fr;
   min-block-size: 100%;
 
   > .background {
-    background-color: awsui.$color-background-layout-main;
-    grid-column: 1;
-    grid-row: 1 / 3;
+    grid-column: 1 / 8;
+    grid-row: 1 / 6;
     color: awsui.$color-text-body-default;
+    &:not(.has-header-background) {
+      background-color: awsui.$color-background-layout-main;
+    }
 
     &.is-overlap-disabled {
-      grid-row: 1 / 2;
+      grid-row: 1 / 5;
+    }
+
+    > .header-background {
+      inline-size: 100%;
+      block-size: 100%;
     }
   }
 
-  > .header {
-    grid-column: 1;
-    grid-row: 1;
+  > .notifications {
+    grid-column: 4;
+    grid-row: 2;
+    padding-block-end: awsui.$space-xs;
+  }
+  > .breadcrumbs {
+    grid-column: 4;
+    grid-row: 3;
+    padding-block-end: awsui.$space-xs;
+  }
+
+  &.default-padding {
+    #{custom-props.$contentLayoutMainGap}: awsui.$space-scaled-s;
+    grid-template-columns:
+      var(#{custom-props.$togglesLeftWidth}, 0) #{awsui.$space-layout-content-horizontal} 1fr minmax(
+        0,
+        var(#{custom-props.$contentLayoutMaxContentWidth})
+      )
+      1fr #{awsui.$space-layout-content-horizontal} var(#{custom-props.$togglesRightWidth}, 0);
+  }
+
+  > .header-wrapper {
+    grid-column: 4;
+    grid-row: 4;
     padding-block-end: awsui.$space-content-header-padding-bottom;
+
+    &.with-divider {
+      border-block-end: 1px solid awsui.$color-border-divider-default;
+    }
   }
 
   > .content {
-    grid-column: 1;
-    grid-row: 2 / 4;
+    grid-column: 4;
+    grid-row: 5 / 8;
   }
 
   &:not(.has-header) {
     grid-template-rows:
-      auto calc(
+      var(#{custom-props.$contentLayoutMainGap}) min-content min-content auto calc(
```

Comment:
We cannot use CSS-vars in content-layout, because it is not widget compatible. I already opened a PR to remove all existing usage https://github.com/cloudscape-design/components/pull/2136 and your change cancels my work